README.md

```diff
@@ -54,6 +54,14 @@ azd init --template Azure-Samples/ASA-Samples-Event-Driven-Application
 azd up
 ```
 
+The template uses [ASA consumption](https://learn.microsoft.com/azure/spring-apps/overview#standard-consumption-and-dedicated-plan) plan by default. If you want to switch to `Standard` plan, you can use the following command before running `azd up`.
+
+```bash
+azd env set PLAN standard
+```
+
+If you have already provisioned the resources with the Consumption plan and want to try the Standard plan, you need to run `azd down` first to delete the resources, and then run the above command and `azd up` again to provision and deploy.
```

Comment:
Confuse for the plan name, below which one is correct?

- consumption plan
- consumption and dedicated plan
- Standard consumption and dedicated plan

Reply:
will use `Standard consumption and dedicated plan`